src/FffPolygonGenerator.cpp

```diff
@@ -135,6 +135,7 @@ bool FffPolygonGenerator::sliceModel(MeshGroup* meshgroup, TimeKeeper& timeKeepe
         //Find highest layer count according to each mesh's settings.
         for(const Mesh& mesh : meshgroup->meshes)
         {
+            if ( ! mesh.isPrinted()) continue;
```

Comment:
lil codestyle
```suggestion
            if ( ! mesh.isPrinted())
            {
                continue;
            }
```

Reply:
Is there some code conventions document? I forgot where to find it.

Reply:
Personally I choose between
```
if (condition) short_statement;
```
and
```
if (condition)
{
  multi
  line;
}
```
based on which would be better readable. Readability is most important.

I avoid the other option in order to prevent this mistake:
```
if (condition)
  part of condition;
  oops;
```

Reply:
code style conventions https://github.com/Ultimaker/Meta/. specifically the codeblock conventions
 https://github.com/Ultimaker/Meta/blob/master/general/generic_code_conventions.md#code-blocks-not-codeblocks-